package.json

```diff
@@ -2,12 +2,12 @@
   "name": "node-i18n",
   "private": true,
   "version": "1.0.0",
-  "nodeVersions": {
-    "v13.x": "v13.11.0",
-    "v12.x": "v12.16.1",
-    "v10.x": "v10.19.0",
-    "v8.x": "v8.17.0"
-  },
+  "supportedVersions": [
```

Comment:
Think it would make sense to drop this concept entirely if we're changing this. Probably better to just scan the content folder for the directory names

Reply:
We need to have some flexibility to control target versions. If we're going to scan content folder:
- no way to fetch v14.x without manual folder creation
- no way to stop fetching deprecated versions v8.x without manual removal of folder